files/__addonLocation__/src/template-registry.ts

```diff
@@ -4,8 +4,7 @@
 
 // import type MyComponent from './components/my-component';
 
-// Remove this once entries have been added! 👇
-// eslint-disable-next-line @typescript-eslint/no-empty-interface
```

Comment:
instead of adding new lint disables here, I've just commented out the whole thing